src/Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis/ConnectionMultiplexerExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using StackExchange.Redis;
+
+namespace Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis
+{
+	internal static class ConnectionMultiplexerExtensions
+	{
+		public static RedisClient GetRedisClient(this ConnectionMultiplexer source, FunctionalRedisCacheConfiguration configuration)
```

Comment:
`StackExchange.Redis` [separates server-based and database-based commands](https://stackexchange.github.io/StackExchange.Redis/KeysScan).  The `RedisClient` object makes it easier for working with existing code.